src/components/NetworkDebuggerModal/index.js

```diff
@@ -20,6 +21,7 @@ const NetworkDebuggerModal = ({onPress}) => {
     setIsNetworkVIsible(true);
   };
 
+  console.log(isDebug);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `console.log(isDebug);` statement seems to be used for debugging purposes and should not be included in the production code. It's a good practice to remove all console logs before committing your code to maintain clean and readable codebase.
```diff
-   console.log(isDebug);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->